web/taskmaster/src/app/taskform.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { MatSelectModule } from '@angular/material/select';
+import { MatInputModule } from '@angular/material/input';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatButtonModule } from '@angular/material/button'
+import { FormsModule } from '@angular/forms';
+import { Task } from './task.model';
+
+@Component({
+    selector: 'app-taskform',
+    standalone: true,
+    imports: [
+        MatSelectModule,
+        MatInputModule,
+        MatFormFieldModule,
+        FormsModule,
+        MatButtonModule,
+    ],
+    templateUrl: './taskform.component.html'
+})
+export class TaskFormComponent {
+    addTask() {
```

Comment:
Usually the order is Inputs, Outputs, Properties, viewchild or similar, constructor, methods based on life cycle hooks, public methods and private methods.